packages/backend/src/controllers/users/dto/user-response.dto.ts

```diff
@@ -1,6 +1,9 @@
 import { ApiProperty } from '@nestjs/swagger';
 
 export default class UserResponseDto {
+  @ApiProperty()
+  name: string;
```

Comment:
```suggestion
  name?: string;
```

should mark as optional

Reply:
my bad 😓 the response should not be optional